### PR TITLE
793 configure simulation run

### PR DESCRIPTION
**Pull request by Derstilon**
image
image

### CODE REVIEW

**Comment by grzanka** [comment]
@Derstilon does it make sense to test it now ?

**Review by grzanka — DISMISSED** [review]
ok

File: src/services/ShSimulatorService.tsx [hunk 1]
```diff
@@ -111,13 +111,23 @@ const ShSimulation = ({ children }: ShSimulationProps) => {
 				console.log('postJob', endPoint, ntasks, simType, title, batchOptions, signal);
```
Comment (ostatni5) [inline src/services/ShSimulatorService.tsx:111]:
```suggestion
```

**Review by ostatni5 — CHANGES_REQUESTED** [review]
Ok but it could be better
image

File: src/WrapperApp/components/Simulation/BatchParametersEditor.tsx [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+	handleScriptOptionsChange: (newValue: ScriptOption[]) => void;
+	scriptName: string;
+};
+
+const _PROPOSED_OPTIONS = [
+	{ optionKey: '-a', optionLabel: 'option a', optionValue: '' },
+	{ optionKey: '-b', optionLabel: 'option b', optionValue: '' },
+	{ optionKey: '-c', optionLabel: 'option c', optionValue: '' },
+	{ optionKey: '-d', optionLabel: 'option d', optionValue: '' }
+] as const;
```
[the remark this answers is not in the document]
Comment (grzanka) [reply src/WrapperApp/components/Simulation/BatchParametersEditor.tsx:34]: `--time=0:15:00`

[the remark this answers is not in the document]
Comment (grzanka) [reply src/WrapperApp/components/Simulation/BatchParametersEditor.tsx:34]: That one should be enough for the beginning

File: src/WrapperApp/components/Simulation/RunSimulationForm.tsx
[the remark this answers is not in the document]
Comment (Derstilon) [reply src/WrapperApp/components/Simulation/RunSimulationForm.tsx:1]: How about now?

**Review by ostatni5 — APPROVED** [review]

**Pull request merged**